### ISSUE TITLE
Translate page titles besides content

### ISSUE THREAD

**Issue by jirichlebus** [johannschopplich/kirby-content-translator#3]
Hello,

I've noticed that it doesn't support the translation of page titles.

This is an essential feature for a multilingual site, as page titles are crucial for both SEO and providing a localized user experience. I attempted several methods, including trying different hooks and methods as suggested in the Kirby documentation, but without success.

Could this functionality be added in a future update?

Thank you for the great work on this plugin.

Best regards,
Jiri

**Comment by johannschopplich** [comment]
Hi there @jirichlebus, the feature is coming. Took a while to get it right. 🙂

**Comment by johannschopplich** [comment]
### Translating the Title

You can enable to synchronise and translate the title of a model by setting the `title` section property to `true`:

```yml
sections:
  contentTranslator:
    type: content-translator
    title: true
```

> [!IMPORTANT]
> Technically speaking, the title is not part of the content of a Kirby model. That's why you can't revert changes made to the title, as opposed to fields. Also, when synchronising content, the title will be overwritten.
> Because of these caveats, have to opt-in to synchronise and translate the title.

**Comment by johannschopplich** [comment]
Closed in https://github.com/johannschopplich/kirby-content-translator/commit/5aebdca04327c2f636bb41f684c782707002ab2d and available in [v1.1.0](https://github.com/johannschopplich/kirby-content-translator/releases/tag/v1.1.0).
